x-pack/plugins/security_solution/server/lib/timeline/saved_object/notes/get_overridable_note.ts

```diff
@@ -5,8 +5,7 @@
  * 2.0.
  */
 
```

Comment:
what is `get overridable note`?

Reply:
It is mostly used by importing timeline with notes. When importing timeline with an existing note by others, we don't want override the owner. In this case we can set overrideOwner to false to keep the original author.